docfx_project/nexus_template/partials/logo.tmpl.partial

```diff
@@ -1,5 +1,6 @@
SV3vMRypEv0uhWmBOBSkvmLLK2f4cDbeOX4COIPtOhHJHc4ZnXOkYtQflcSyOWJx8MRKk+X6f9SaAReb9mE7Oq9kvwwZjWRBFjX2CHToVv81d7K8SpxWowGh2ddgdo/kXr+1mP/NXPxCmMX4JHEc9lHMPZOgbbfgBt5+O/oZ6ZruudQOfjizDax7Kq30Xa5t24BPFrRcd100oNcoLkg1rU9OWJmGTrDseiJPpKy6rOIHvhJhS0iByJDRcalJCHAyVnP54BvOVacORKvRBjjFr78SZk+f+eGLxxSSe4ATUd734gbSQ/gTxIa1wLjzTQFHTqSFn78Roj4z+iFsAvwXHtcx9TQCqM5QS5E3gA901fUugsfzVb+6s9bfNmlAL4IbgFXcQRR4X+cvxZxhuA61HSnmujqwQ4FKnr/jn+6mcRyZFlZvoleCHxrI43KbDJeE5R240kZz8+EziS9ufjJmT930NE/mq/BD9C9GfOjcjiLLgLgEXyfNRpLwmdfYahS7z2hHaqeh3KypwXxUP9ErzECBSlmp6H+nh0mq/V5cDaPnM3yniM5VjSBTwX73RgWI6qXoKs6tD344IJtrrXzQI+IBoHeh2yMAPVDFtBiRvQftccgayFoBTtx5cDA3NInofm9OMwHxjEin4Nud4+DXkSGtG+9CT4W73tBqjPv48s2XdwnyRQibJXjsPsx4bkLG3RsdCCEkHOwY3INTg7RIG8vh23ACuDkts6yGmrvCLz60lGZ58a1GlvArSLH29APvXnCckJEuRyStBkTUWqpS6gLJuBh833vRkWuRaaUUHb7bg/H4OMrfOB3by/MCS/gxIZlhCCtgl68xmI5OnIib7E59etRd12fkU05HrYjFbIg7hX1SXAYSgrc9ucmPdTyK++loCZr2EmvleifOiJKOVm124+1oKOCE+iY9djwNoIyfXkTKNEgpnmV9f4EB2fZgPNVn51FdJmxyJvmLvKBg9mAquAvVFd7T4otLgdSgJvMQNahSobXjc/70dJbAcy9kMv4UzCq0AIgkXI09VqWBqSRyPNeDrwqXOCPVh5W0NRInwlbSHGemR5rwE2xkVsB/INQi0ariS+8GdXeBzllr/lzUk6XZJGt7gtApoTQ3DSYe13O6Boz2muZUJG4K1oxdZaqziFsReKBBcAi+QxqIf1Qa5lQsfN89B5uBGC+wFCJ9iauBLz4zUq8d5Er2Q04/3ehZrOkfUY5LQZ4USQ9liG7kB+Noy5CY1gq5ylH1AN7IeyC0egxPDBtFUTLkf3KT2DQmWbwxhMALnL0Jl0GsnoU/kUys58G4Kt4rDKR0GrdRTwPdTPYjRtq9hbyaCVm0F7Th1yPtyJEtE2x02yJX8Nqos6JVYBOkYGuAPVfdWCf5LDKB8FWcmHAVegI5JNaD6D+QAFBP6M8YjFSbQ1jrHInXlwbA/vHI3o6HQHJm3JV1QthEmpQtfG3YvymEsp7MUpQYVmN6DJHV7AZ0OBNXEvo+LtJCTRlyELfzwBjnFBykdBNUSnor1rIME0wgCUZjoF2D7u1oEWyQ8hf/W6WAXoGNXo2DQcn3MbJJpUipqiTCW8avsK4GTgx5j8pThhlafOQkS7Dkp4VZwn4bP+2i/BKeRrnko3lQc+UI76W7TW28YJQ3I9iui8iPukvXKkJYf5mQu/BPdDSWS7EY2zZDSqnN8+gu/uflJE8iLkAFnuQgYLKeTLPwQfc+23PngH1L8xyjPjkbi/BeVhlHxQ50oAgwpEcOEpVj4elkYOjOqIB1WNCM63NilUWDe13YmOb3lVWESEchSZK3wcPj6TQn7bOMpHx+BITUO7/fhG5Dp02dlnMFBRqDbzS/BexOPSG0l0bSLygiF5BUraW4a7TJAyOmhM0638Ph82IKZBOSfYIIuS9q5FiXEu0IKP6oekVxdWImdKEtCIcrlm4UZVNwIrC/1Q0glOBCwv13pUCjOPeFV1FqU5NRXqj/ZLcFyDW04y8phtklcg9+HiGB/fBPzPl9w+PtOCVEUcHp71qCI+ScgCC1AuV1z+6o2oP2fBc+6X4AXEcy5cRIHlo1HCWsVNKAPyZqKvk/YW1L/woTn9EjyPeKr1XgY+iuE5ecMiuQ7lTj0Q8SO3oOpIX+W6fqoLW1C6zWKiVdMLgVeArKucrS7mwCP6PVQp8UZEj/KK/G7Dh4EF/o2sJlTPGnZloYdm1AE+qokLEwuA61BWStj4CDlYfBuafgnOoHjpA0RTc/ssIthJYny+sPzVD6KgRJgW/4eon9bTmJIWXzIGEGATOi68QLjHJu9CrEi7z4QFM/EbUSeBawhnJb+HtMLtwCYnWZXWRRnjUXVhGIlqb6NMwvsJ8Na6gJV+ezRqtTyewr1wGeQSnYk0WIPTvGgzqFIU9TnHDM5PeK/JDOw6dPzwZVAkAWZO9kTFbZNQyLO7wEwGGVNz0eUkbwGNYcxBmNfqDEcVAmegaFM+8M54c1C3gDd6KrEdzEkFarqyB7A/ujllFG2Bmk2otcQKZEy+gogN1e4Iu7JhAGqjOwY1GxmLQlx24/AWtE8tRPv3S2ZgkdYGu4K557Da/FRZc9GEIlN16IwbyZYUVW1SGeoLNQyp7DQifwtSR58hi/MTQthniiiiiCKKKKIH4v/avnpHFNef3gAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOC0wNC0xM1QxMzoxMzoxOCswMjowMNsMZJUAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTgtMDQtMTNUMTM6MTM6MTgrMDI6MDCqUdwpAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAFd6VFh0UmF3IHByb2ZpbGUgdHlwZSBpcHRjAAB4nOPyDAhxVigoyk/LzEnlUgADIwsuYwsTIxNLkxQDEyBEgDTDZAMjs1Qgy9jUyMTMxBzEB8uASKBKLgDqFxF08kI1lQAAAABJRU5ErkJggg==" alt="{{_appName}}" >
```

Comment:
@andrea-borzi You should change this part for two reasons

1. You should avoid inlining binary data for an image: place the stuff somewhere
2. Why you're not using the SVG?